integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/BenchmarkRunner.scala

```diff
@@ -95,6 +100,24 @@ object BenchmarkRunner {
               conf.iterations(),
               summaryFilePrefix = conf.summaryFilePrefix.toOption,
               gcBetweenRuns = conf.gcBetweenRuns())
+        })
+
+        report match {
+          case Success(report) =>
+            if (conf.uploadUri.isSupplied) {
+              println(s"Uploading ${report.filename} to " +
+                  s"${conf.uploadUri()}/${report.filename}")
+
+              val hadoopConf = spark.sparkContext.hadoopConfiguration
+              val fs = FileSystem.newInstance(new URI(conf.uploadUri()), hadoopConf)
+              fs.copyFromLocalFile(
+                new Path(report.filename),
+                new Path(s"${conf.uploadUri()}/${report.filename}"))
```

Comment:
Probably best to use the existing `Path(parent, child)` constructor here to handle cases where the parent ends in a path terminator already.
```suggestion
                new Path(conf.uploadUri(), report.filename))
```